ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -116,6 +116,37 @@ def test_numpy_ndarray_property_T(
     )
 
 
+import traceback
+
+
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="flat",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes(kind="valid"),
+    ),
+)
+def test_numpy_instance_flat(
+    dtypes_and_x, frontend_method_data, init_flags, method_flags, frontend, on_device
+):
+    input_dtypes, vals = dtypes_and_x
+    print(type(vals))
```

Comment:
Will this be removed later

Reply:
Yess! I was just doing some debugging :). I will remove it and commit the change.